Bonsai.Tinkerforge/BrickletIndustrialPTC.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.ComponentModel;
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using Tinkerforge;
+
+namespace Bonsai.Tinkerforge
+{
+    [DefaultProperty(nameof(Uid))]
+    [Description("Measures temperature with Pt100 and Pt1000 sensors from a PTC Industrial Bricklet")]
+    public class BrickletIndustrialPTC : Combinator<IPConnection, int>
```

Comment:
This is just a clarification comment. In this case we will opt to leave the acronym PTC capitalized, since it matches the underlying Tinkerforge API, so it is fine as it is.